python/lancedb/pydantic.py

```diff
@@ -249,3 +249,17 @@ def pydantic_to_schema(model: Type[pydantic.BaseModel]) -> pa.Schema:
     """
     fields = _pydantic_model_to_fields(model)
     return pa.schema(fields)
+
+
+class LanceModel(pydantic.BaseModel):
+    """A Pydantic Model that can be converted to a LanceDB Table."""
```

Comment:
So user exepct to inherent this class or take this as a Mixin?
Can we add an explicit example for that?